.changeset/slimy-jobs-smash.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+Fix dot files not being copied over on non-root builds
```

Comment:
```suggestion
Fixes an issue where dot files were not copied over from the public folder to the output folder when build command was run inside a subfolder.
```

Reply:
It's not quite just a subfolder, it's any folder that is not the root of the project. I've updated the changeset description to reflect this.